packages/mattermost-redux/src/client/client4.ts

```diff
@@ -1006,6 +1008,13 @@ export default class Client4 {
         );
     }
 
+    getStatusBasedOnSchedule = (userId: string, currentTime: string, currentDay: string) => {
```

Comment:
What happens to places which gets status directly from /status endpoint? Correct me if I am wrong but they could be different from status gotten from this endpoint, right?

Reply:
> What happens to places which gets status directly from /status endpoint? Correct me if I am wrong but they could be different from status gotten from this endpoint, right?

we can get user's status (e.g. online, away, dnd ...) directly from /status endpoin. 
but on our approach, we need to call the API periodically (every 1 minutes). So we made our own.